src/platform/packages/shared/kbn-es-query-server/index.ts

```diff
@@ -19,4 +20,11 @@ export type {
   FilterMeta,
   AggregateQuery,
   Query,
+  SimplifiedFilter,
+  SimpleFilterCondition,
+  FilterGroup,
+  RawDSLFilter,
+  FilterValue,
+  RangeValue,
+  StoredFilterState,
```

Comment:
In a similar vein, while maybe redundant, would it be worth prefixing all related types with `Simple` to distinguish them from the legacy types?